motr/m0crate/parser.c

```diff
@@ -116,8 +118,6 @@ struct key_lookup_table lookuptable[] = {
 	{"OPS", NUM_OPS},
 	{"NUM_IDX", NUM_IDX},
 	{"NUM_KVP", NUM_KVP},
-	{"RECORD_SIZE", RECORD_SIZE},
```

Comment:
Fixed.

Reply:
Record size was actually key-size+value-size. 

Since we are updating all test yaml files in this PR, how about just updating yaml files to remove RECORD_SIZE and use the new params. This way code does not have old traces of record size.

Reply:
I think we can remove the "RECORD_SIZE" from yaml config files. But, let's keep this "keyword" here.
Other developers may have their own yaml config, and removing this keyword from released version will affect their scripts.

Reply:
We are now keeping RECORD_SIZE and MAX_RSIZE, so m0crate will work for old yaml sample files, please review the implementation.